modules/git/init.zsh

```diff
@@ -45,9 +45,8 @@ alias gcP='git cherry-pick --no-commit'
 alias gcr='git revert'
 alias gcR='git reset "HEAD^"'
 alias gcs='git show --pretty=format:"${_git_log_medium_format}"'
-alias gcl='git-commit-lost'
 alias gcS='git commit -S'
-alias gpS='git show --pretty=short --show-signature'
+alias gcv='git verify-commit'
```

Comment:
I've changed from `git show --show-signature` to `git verify-commit` because we're already using `git verify-tag` at `gtv`.